crates/networking/p2p/rlpx/message.rs

```diff
@@ -38,6 +46,14 @@ impl Message {
             Message::Ping(msg) => msg.encode(buf),
             Message::Pong(msg) => msg.encode(buf),
             Message::Status(msg) => msg.encode(buf),
+            Message::GetAccountRange(msg) => {
+                0x21_u8.encode(buf);
```

Comment:
this is quite manual. Would be nice to refactor code so that a message is associated to a message id (probably out of scope of this PR).

Reply:
We could add a method `code` to the `Message` enum and use it for encoding/decoding so that we onlty write the code number once in the codebase

Reply:
Sounds like a good solution.

Reply:
Ill create an issue for it

Reply:
#1035

Reply:
I noticed that the msg_id is encoded within the `RLPxMessage::encode(..)` for the other `Message` types like `PingMessage`, `HelloMessage`. Would it not work if we move `0x21_u8.encode(buf)` into the `RLPxMessage::encode(..)` method of `snap::GetAccountRange`?

Reply:
We are discussing moving the encoding of the code number to the encoding of the `Message` enum for the other messages too. The code number is already being decoded on the enum's decode `Message` so encoding it on the enum's decode method would keep encoding/decoding simmetrical for all involved structs